00-types.types

```diff
@@ -28,6 +28,9 @@
 # Type: Chat
 { "type": "Chat", "nice": -3, "ioclass": "best-effort", "ionice": 7 , "latency_nice": -3 }
 
+# Type: Compiler
+{ "type":"compiler", "nice": 9, "latency_nice": 9 }
```

Comment:
Should we maybe put it a bit higher like 11 to 13?
@aarrayy WDYT?

Reply:
I have been daily driving this at 13 for two weeks now, it is stable. 13 should leave some more headroom.